docs/content/querying/sql.md

```diff
@@ -571,6 +571,8 @@ The "sys" schema provides visibility into Druid segments, servers and tasks.
 ### SEGMENTS table
 Segments table provides details on all Druid segments, whether they are published yet or not.
 
+#### CAVEAT
+Note that a segment can be served by more than one realtime or historical servers, in that case it would have multiple replicas. These replicas are weakly consistent with each other when served by multiple realtime tasks, until a segment is eventually served by a historical, at that point the segment is immutable. And broker prefers to query a segment from historical over realtime server. But if a segment has multiple realtime replicas, for eg. kafka index tasks, and one task is slower than other, then the sys.segments query results can vary for the duration of the tasks. The columns of segments table that can have inconsistent values during this period include `size`, `num_replicas`, `num_rows`.
```

Comment:
There are no such things are realtime or historical servers

Reply:
- please ensure consistent capitalization for Historicals, Brokers, etc

Reply:
> There are no such things are realtime or historical servers

I see mention of `Historical Node`, `Real-time Node` in docs. So what should I write historical node ? process ?

Reply:
IMO, "Historical process" and "ingestion tasks"